functions/apiProxy/main.py

```
@@ -103,14 +120,19 @@ def handle_moochub_data(page=1, per_page=25):
         transformed_data = []
         for course in courses["data"]["Course"]:
-            # Determine courseMode - "online" if ONLINE exists, otherwise "onsite"
-            course_mode = ["online"] if any(
-                loc["locationOption"] == "ONLINE" 
-                for loc in course["CourseLocations"]
-            ) else ["onsite"]
+            # Collect metadata tags (sliderGroup = false)
+            metadata_tags = []
+            for group in course.get("CourseGroups", []):
+                group_option = group.get("CourseGroupOption", {})
+                if group_option.get("sliderGroup") == False:
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not group_option.get("sliderGroup"):` for false checks

Replace with `not group_option.get("sliderGroup")`

(E712)